### ISSUE TITLE
SquareSet.issubset() and SquareSet.issuperset() do opposed things

### ISSUE THREAD

**Issue by drondd** [niklasf/python-chess#838]
```
>>> import chess
>>> chess.SquareSet(chess.BB_RANK_1).issuperset(chess.SquareSet(chess.BB_A1))
False
>>> chess.SquareSet(chess.BB_RANK_1).issubset(chess.SquareSet(chess.BB_A1))
True
```
They should do opposed thing, just like:
```
>>> {1,2,3}.issuperset({1,2})
True
>>> {1,2}.issubset({1,2,3})
True
```

**Comment by niklasf** [comment]
Ouch ... thanks for reporting. This one hurts badly, because I would usually rename these methods to force people to acknowledge the issue (in case they already rely on the broken behavior), but it's not possible while "keeping" ducktyping compatibility with `set`.

**Comment by niklasf** [comment]
Published as v1.8.0.
